website/content/docs/v0.15/Single Board Computers/jetson_nano.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Jetson Nano"
+description: "Installing Talos on Jetson Nano SBC using raw disk image."
+---
+
+## Prerequisites
+
+You will need
+
+- `talosctl`
+- an SD card/USB drive
+- [crane CLI](https://github.com/google/go-containerregistry/releases)
+
+Download the latest `talosctl`.
+
+```bash
+curl -Lo /usr/local/bin/talosctl https://github.com/talos-systems/talos/releases/latest/download/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
+chmod +x /usr/local/bin/talosctl
+```
+
+## Flashing the firmware to on-board SPI flash
+
+We will use the [R32.6.1 release](https://developer.nvidia.com/embedded/l4t/r32_release_v6.1/t210/jetson-210_linux_r32.6.1_aarch64.tbz2) for the Jetson Nano.
```

Comment:
I don't know if it's true, but I guess flashing firmware is one-time procedure? we don't need to flash it every time?

(does it makes sense to state it in the docs?)

Reply:
good catch, fixed